packages/sdk/test/flow.spec.ts

```diff
@@ -260,6 +261,53 @@ describe('Flow Application', () => {
 
     expect(loggerMock.log).toHaveBeenCalledWith('Flow Deployment is running', expect.objectContaining(flow.context));
   }, 60000);
+
+  it('FLOW.FA.9 test complex properties', (done) => {
```

Comment:
the test succeeds wether enableImplicitConversion is set to true or false

Reply:
thats correct for some reason it always works in this repo, but in the hpc it does not work without the parameter

Reply:
The reason was that the wrong class-transformer version was used in the flow modules repo. Installing with the "legacy-peer-deps" flag resolved it.
You can remove the workaround if you want to merge the test anyways

Reply:
Can you please also remove the changeset and update the PR title?